terms.html

```diff
@@ -22,47 +22,48 @@
 <p><span class="section-number">5</span>We may choose to annotate <em>this agreement</em> to explain why we have chosen to draft it in this way. If we do so, those annotations will not form a part of this contract or have contractual force.</p>
 <h2>Our commitment to you</h2>
-<p><span class="section-number">6</span>We will provide access to <em>the platform</em> as detailed in <em>the pricing plan</em>, and subject to the limitations (bandwidth, storage, etc.) contained in that plan.  We will exercise reasonable care and skill in providing <em>the platform</em>, and will do what is reasonable in the circumstances to ensure that <em>the platform</em> is operational as far as is possible.</p>
+<p><span class="section-number">6</span>We will provide access to PDF Tables as detailed in <em>the pricing plan</em>, and subject to the limitations (eg. as to file size or number of files)contained in that plan.  </p>
+<p><span class="section-number">6</span>PDF's can be complicated files. It is impossible for us to guarantee that the tables extracted from a PDF will be accurate. Some PDF's may not work at all. While we hope that PDF Tables will work well for you, we make no promises as to its functioning.</p>
+<p><span class="section-number">6</span>Your use of the service will be governed by your <em>payment plan</em>. If you have not signed up to a particular plan (for example if you are a casual user of the website) then your use will be governed by one or more default <em>payment plans</em> selected by us (see: <a href="http://pdftables.com/pricing">http://pdftables.com/pricing</a>)</p>
 <p><span class="section-number">7</span>No guarantee as to availability or performance is offered.  If you require a guaranteed service level, please contact us to discuss pricing and availability.</p>
-<p><span class="section-number">8</span>We will not access any <em>content</em> which you store on <em>the platform</em>, including your contact details, except in order to:</p>
+<p><span class="section-number">8</span>We will not access any <em>content</em> which you store on PDF Tables, including your contact details, except in order to:</p>
 <ul>
   <li><span class="section-number">8.1</span>Provide and administer the services in <em>the pricing plan</em>;</li>
-  <li><span class="section-number">8.2</span>Improve and maintain <em>the platform</em> or any tools that we supply for use with <em>the platform</em>; </li>
+  <li><span class="section-number">8.2</span>Improve and maintain PDF Tables or any tools that we supply for use with PDF Tables; </li>
```

Comment:
We also sometimes use the PDFs for market segmentation - e.g. to identify there is a use case in Insurance. So we can then market to those segments, or develop special features for them. Is that within these clauses, or does it need separately mentioning?